contracts/contracts/TestValidatorAuctionFixedPrice.sol

```diff
@@ -9,6 +9,8 @@ import "./ValidatorAuction.sol";
 
 
 contract TestValidatorAuctionFixedPrice is ValidatorAuction {
+    constructor(DepositLocker _depositLocker) ValidatorAuction(_depositLocker) {
```

Comment:
I'd rather we do it the way it is done in Bernd's PR: https://github.com/trustlines-protocol/blockchain/pull/25/files#diff-ec429f556412759d5ab5e44787fb4a82R11
Mostly for consistency but also because it is the recommanded way

Reply:
Can you please write that down for me? I don't know how this is supposed to be done. Bernds PR only uses constant values. I don't know how to pass that argument down to the base constructor.

Reply:
You're right, it actually does not work, sorry.